python/tvm/relay/_parser.py

```diff
@@ -114,7 +114,10 @@ def convert(self, v):
     def __call__(self, args, attrs, type_args):
         if attrs is None:
             attrs = {}
-        x = self.operator(*args, **{k: self.convert(v) for k, v in attrs.items()})
+        if self.operator is op.reshape:
```

Comment:
Does this mean we have to maintain a list for symbolic ops here?

Reply:
I think for now we might want to handle each symbolic op separately, since they may have different attrs.

Reply:
Do we still this special handling after using ```Optional```?

Reply:
Do we still need this after using "Optional"?